app/src/test/java/com/kickstarter/ui/activities/compose/LoginToutScreenTest.kt

```diff
@@ -89,6 +89,7 @@ class LoginToutScreenTest : KSRobolectricTestCase() {
         touPpCookieDisclaimer.assertIsDisplayed()
     }
 
+    @Test
```

Comment:
This test didnt have the tag and wasnt running before, still works, but now it runs